cloud/aws/ecs.md

```diff
@@ -3,7 +3,9 @@ layout: page
 title: "Amazon ECS with Portworx"
 keywords: portworx, amazon, docker, aws, ecs, cloud
 sidebar: home_sidebar
-redirect_from: "/portworx-on-ecs2.html"
+redirect_from:
+  - /portworx-on-ecs2.html
+  - /cloud/aws-ecs.html

```

Comment:
You're thinking of redirecting this page (https://docs.portworx.com/portworx-on-ecs.html) to Amazon ECS with Portworx?